dpgen/generator/lib/modd.py

```diff
@@ -0,0 +1,122 @@
+import argparse
+import copy
+import dpdata
+import glob
+import math
+import numpy as np
+import os
+import sys
+import shutil
+
+
+def write_model_devi_out(devi, fname):
+    assert devi.shape[1] == 8
+    #assert devi.shape[1] == 7
+    header = "%5s" % "step"
+    for item in 'vf':
+        header += "%16s%16s%16s" % (f"max_devi_{item}", f"min_devi_{item}",f"avg_devi_{item}")
+    header += "%16s"%str('min_dis')
+    np.savetxt(fname,
+               devi,
+               fmt=['%5d'] + ['%17.6e' for _ in range(7)],
+               delimiter='',
+               header=header)
+    return devi
+
+
+def Modd(all_models,type_map):
```

Comment:
It seems that `Modd` only works for the case of calypso. Please move it to module `model_devi_calypso`